cedar-policy-cli/tests/sample.rs

```diff
@@ -792,9 +792,7 @@ fn test_link_samples() {
         &linked_file_name,
         "AccessVacation",
         "AliceAccess",
-        [(SlotId::principal(), "User::\"alice\"".to_string())]
-            .into_iter()
-            .collect(),
+        std::iter::once((SlotId::principal(), "User::\"alice\"".to_string())).collect(),
```

Comment:
Here and else where: I think these could be `HashMap::from([...])` instead going through an iterator. Or maybe `run_link_test` could just take an `impl Into<HashMap<_,_>>`